README.md

```diff
@@ -51,7 +51,20 @@ We are always trying to improve our documentation. If you have suggestions or ru
          * Read a [Windows tutorial](http://www.postgresqltutorial.com/install-postgresql/)
          * Read a [Linux tutorial](https://www.postgresql.org/docs/9.4/static/installation.html) (or follow your OS package manager)
     * Elastic Search 5.6 (instructions [here](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/_installation.html))
-    * Flyway 5.0.x ([download](https://flywaydb.org/getstarted/download))
+    * Flyway 5.2.4 ([download](https://flywaydb.org/getstarted/download))
+		* After downloading, open `flyway5.2.4/conf/flyway.conf` and set
+           the flyway environment variables `flyway.url` and
+           `flyway.locations` as
+		   
+		   ```
+		   flyway.locations=filesystem:/Users/<user>/<project>/api/openFEC/data/migrations
+		   flyway.url=jdbc:postgresql://localhost:5432/cfdm_test?user=&password=
```

Comment:
Great work @jason-upchurch! Should the instructions say `user=<your username>&password=<your username>`?

Reply:
Agreed @lbeaufort. I'll make the update now.